### PR TITLE
Implement/redis

### PR DESCRIPTION
**Pull request by Busyreadingsomething**
[IMPLEMENT REDIS](https://trello.com/c/UeatYANu/37-redis-m)
[REFACTOR REDIS](https://trello.com/c/bm4y12Zk/45-refactor-redis-code-m)
[MOCKS](https://trello.com/c/TOQNObBq/44-improve-testing-with-mocks-m)
@nicola24 Thanks again man <3

### CODE REVIEW

**Review by nicola24 — COMMENTED** [review]
Circle CI failling

**Comment by Busyreadingsomething** [comment]
@mlbpro417 Thanks dude <3

File: server/helpers/redisCache.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const redisClient = require('./redisClient');
+
+const checkCache = (req, res, next) => {
+  redisClient.get('bundle', (err, data) => {
+    if (err) {
+      console.log('Failed to get from Redis', err);
+      next();
+    } else if (data != null) {
```
Comment (mlbpro417) [inline server/helpers/redisCache.js]: should this be !==

**Review by mlbpro417 — APPROVED** [review]
nice job

**Review by mlbpro417 — APPROVED** [review]

**Pull request merged**